include/picongpu/plugins/radiation/radFormFactor.hpp

```diff
@@ -142,15 +142,41 @@ namespace picongpu
                         /* currently a fixed sigma of DELTA_T * c is used to describe the distribution - might become a
                          * parameter */
                         return math::sqrt(
-                            N
-                            + (N * N - N)
-                                * util::square(
-                                    math::exp(float_X(-0.5) * util::square(omega * float_X(0.5) * DELTA_T))));
+                            N + (N * N - N) * math::exp(float_X(-1.0) * util::square(omega * float_X(0.5) * DELTA_T)));
                     }
                 };
             } // namespace radFormFactor_Gauss_spherical
 
 
+            namespace radFormFactor_Gauss_spherical_simple
+            {
+                struct radFormFactor
+                {
+                    /** Form Factor for point-symmetric Gauss-shaped charge distribution of N discrete electrons:
+                     * \f[ <rho(r)> = N*q_e* 1/sqrt(2*pi*sigma^2) * exp(-0.5 * r^2/sigma^2) \f]
+                     * with sigma = 0.5*c/delta_t (0.5 because sigma is defined around center)
+                     * simplified to avoid calling too often special functions to:
+                     * sqrt( N + (N^2 - N) * F(omega) )
+                     * F(omega) = ( exp(-0.5 * (x/2.0 * delta_t)^2) )^2 ~ 1/(0.03*(delta_t)^2 * x^2 + 1)^12
```

Comment:
I would write ``~ 1/(0.025*(delta_t)^2 * x^2 + 1)^12`` as in the implementation further below. Based on which criterion did you choose the factor 0.025? Please document this because otherwise someone will search for meaning later, where there might be none.

Reply:
Documentation could be improved, although it is "fairly obvious" that the simplification is basically just a series expansion of `exp` (to 4th order).

Reply:
Well, it is not obvious whether you optimize for the minimum relative error integrated over the full spectrum for a given exponent (as Richard did) or if you do something different. You could integrate the absolute errors instead, try to minimize the maximum relative or absolute deviation or let's say compare the integrals of the form factor functions. All these metrics can be justified for all practical purposes considered here, but it is useful to document the metric, so that one does not have to do any forensic code reproduction later.

Reply:
Since including graphs etc is not easily possible in the code and adding it in the radiation plugin documentation would be an overkill for one specific form factor, I referenced the PR id for those whowants to read about the details of the optimization.